### PR TITLE
Change SelinuxPolicy to SelinuxProfile

### PR DESCRIPTION
**Pull request by JAORMX**
#### What type of PR is this?

/kind cleanup
/kind api-change

#### What this PR does / why we need it:

SelinuxPolicy was always a little different when it shouldn't be. In order to
standardize on naming and make this a more cohesive operator, this changes the
name of the aforementioned object to SelinuxProfile. Thus matching
`SeccompProfile` which was supported from the start.

#### Which issue(s) this PR fixes:

None

#### Does this PR have test?

N/A.

#### Does this PR introduce a user-facing change?

```release-note
SelinuxPolicy has been removed and is now SelinuxProfile.
```

### CODE REVIEW

File: internal/pkg/daemon/selinuxprofile/setup.go [hunk 1]
```diff
@@ -38,14 +38,14 @@ func Setup(ctx context.Context, mgr ctrl.Manager, l logr.Logger) error {
 	if err != nil {
 		return errors.Wrap(err, "creating policy wrapper template")
```
Comment (saschagrunert) [inline internal/pkg/daemon/selinuxprofile/setup.go:39]:
```suggestion
		return errors.Wrap(err, "creating profile wrapper template")
```
also renaming the above `policyWrapper` to `profileWrapper`?

File: internal/pkg/daemon/selinuxprofile/utils.go [hunk 2]
```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package selinuxpolicy
+package selinuxprofile
 
 // NOTE(jaosorior): We can probably use a library for generating names instead
 // of using this.
```
Comment (saschagrunert) [inline internal/pkg/daemon/selinuxprofile/utils.go:20]: What about changing `GetPolicyK8sName` within this file, too?

**Review by saschagrunert — COMMENTED** [review]
Do we wanna change `examples/selinuxpolicy.yaml`, too?

**Comment by JAORMX** [comment]
@saschagrunert thanks for the review! Addressed the comments.

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**